### DISCUSSION TITLE
New Default GMod Map

### DISCUSSION THREAD

**Discussion by Derpius** [TAServers/issues#15] [Polls]
As discussed [here](https://github.com/orgs/TAServers/discussions/9), the current default map of `gm_construct` isn't everyone's favourite.

While there's no map that will please everyone, we can probably find a better fit for the server's default.

If you have a different suggestion than what's been put below, feel free to post it here or in Discord and I'll add it to the list.

**Comment by Kilometres** [comment]
i think `gm_bigcity` and `gm_bigcity_improved` should have their votes counted together, and if it wins then the winner of the two should be chosen.
To prevent a case where 60% of plrs want a variation of bigcity (30% normal, 30% improved) but another map is chosen for 40%.

**Reply by Derpius** [reply]
I'll aggregate afterwards (easier to remove data than add it back in)

**Comment by Kilometres** [comment]
Since `gm_bigcity` variants won, vote for `gm_bigcity` or `gm_bigcity_improved`?

**Reply by Derpius** [reply]
Only 4 votes but looks like this’ll be the case. I’ll post again in the discord and give it until Saturday though

**Comment by ELPatitoLoco** [comment]
Why don't we end the pull quicker by making public vote in discord? I don't think to much people is gonna bother to get into github nor a old threatd

**Reply by Kilometres** [reply]
Yeah, maybe just an announcement people react with numbers for each map (no point getting a poll bot just for this and we don't have polls frequently)

**Reply by ELPatitoLoco** [reply]
Thats the idea, just emojis

**Reply by Derpius** [reply]
I’ll do that with the maps that received a vote

Worth noting that there’ll be people in gmod without discord too, so ideally we’d have a persistent poll system in gmod.

YAGNI for this but ill probably add it to the contributions discussion

**Comment by Kilometres** [comment]
Just a reminder that this should also effect [the curtime restart map](https://github.com/TAServers/gmod-utils/blob/master/lua/sv-tas-utils/curtimerestarter.lua)

**Comment by Derpius** [comment]
Poll is taking place in Discord
